indicate when one should update BAM, MBI and MAP with links (#3812)
centreon/centreon-documentation: versioned_docs/version-24.10/upgrade/upgrade-from-22-04.md

### Upgrade the Centreon solution

1. Make sure all users are logged out from the Centreon web interface before starting the upgrade procedure.

> 2. If you have installed Business extensions, delete the configuration of repository 21.10 : 

Author: ```suggestion
2. If you have installed Business extensions, delete the configuration of repository 22.04 : 
```

<Tabs groupId="sync">
<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">

```shell

> rm /etc/yum.repos.d/centreon-business-21.10.repo

Author: ```suggestion
rm /etc/yum.repos.d/centreon-business-22.04.repo
```

```

</TabItem>

<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">

```shell

> rm /etc/yum.repos.d/centreon-business-21.10.repo

Author: ```suggestion
rm /etc/yum.repos.d/centreon-business-22.04.repo
```

  ![image](../assets/upgrade/web_update_5.png)

> > Refer to the [Centreon BAM](../service-mapping/update.md) and [Centreon MAP](../graph-views/map-web-upgrade.md) dedicated procedures to update these modules.

Author: ```suggestion
> Refer to the [Centreon BAM](../service-mapping/update.md), [Centreon MBI](../reporting/update.md)  and [Centreon MAP](../graph-views/map-web-upgrade.md) dedicated procedures to update these modules.
```
